CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Improved console output in verbose mode.
 * Improved logging from multiprocessing jobs.
 
+### Removed
+
+* `umi_unique_count` and `upi_unique_count` from `edgelist`.
+* `normalized_rel` and `denoised` from `obsm` in `anndata`.
+* `umi`, `median_upia_degree`, `median_umi_degree` and `median_reads`  from `component` metrics.
```

Comment:
Very small thing: I would just add "Remove " here before each entry to keep with the changelog style.